neurots/schemas/distributions.json

```
@@ -49,6 +49,10 @@
                     "$ref": "#/definitions/data_distr",
                     "description": "The data distribution used for the number of trees to synthesize"
                 },
+                "min_bar_length": {
+                    "type": "number",
```

It's also possible to add a min value because I guess it can not be `<0`
```suggestion
                    "type": "number",
                    "minimum": 0,
```
You can also add `"exclusiveMinimum": true` if it can not be `<=0`.

---

Good idea, thanks!